### ISSUE TITLE
heap don't show thread's heap

### ISSUE THREAD

**Issue by pullp** [pwndbg/pwndbg#443]
### Description
recently I use pwndbg to dbg a program which call malloc in a thread, but after malloc, I use heap to
show the heaps and I don't see the chunk malloced by the thread, only the chunks in the main arena

### Steps to reproduce
here's the c source code
```c
#include <stdio.h>
#include <stdlib.h>
#include <pthread.h>
#include <unistd.h>
#include <sys/types.h>

void* threadFunc(void* arg) {
        void *ptr;
        ptr = malloc(0x10000);
        printf("11ptr: %p\n", ptr);
}

int main() {
        pthread_t t1;
        void* s;
        int ret;
        ret = pthread_create(&t1, NULL, threadFunc, NULL);
        if(ret)
        {
                printf("Thread creation error\n");
                return -1;
        }
        ret = pthread_join(t1, &s);
        if(ret)
        {
                printf("Thread join error\n");
                return -1;
        }
        return 0;
}
```

I first set a breakpoit at  `printf("11ptr: %p\n", ptr);` in threadFunc.
then run to the breakpoint
then use heap instruction, but I get following answer:
```
Top Chunk: 0x602120
Last Remainder: 0

0x602000 PREV_INUSE {
  prev_size = 0x0, 
  size = 0x121, 
  fd = 0xf, 
  bk = 0x0, 
  fd_nextsize = 0x1, 
  bk_nextsize = 0x0
}
0x602120 PREV_INUSE {
  prev_size = 0x0, 
  size = 0x20ee1, 
  fd = 0x0, 
  bk = 0x0, 
  fd_nextsize = 0x0, 
  bk_nextsize = 0x0
}
```
which don't include chunk just malloced in the threadFunc

version info:
>Gdb:      GNU gdb (Ubuntu 7.11.1-0ubuntu1~16.5) 7.11.1
Python:   3.5.2 (default, Nov 23 2017, 16:37:01)  [GCC 5.4.0 20160609]
Pwndbg:   1.0.0 build: 99e5b84
Capstone: 4.0.1024
Unicorn:  1.0.1

**Comment by disconnect3d** [comment]
Hey! Thanks for reporting the bug. I just have checked and indeed, there is a problem.

I am not really into the heap stuff and its implementation in pwndbg that much. Would you like to try fixing this?

**Comment by disconnect3d** [comment]
Fixed with PR #479.
